release-notes/SRM/SRM_1.2.34_CU1.md

```diff
@@ -0,0 +1,20 @@
+---
+uid: SRM_1.2.34_CU1
+---
+
+# SRM 1.2.34 CU1
+
+> [!NOTE]
+> This version requires that **DataMiner 10.3.2.0 – 12627 or higher** is installed. It is not compatible with the DataMiner Main Release track.
+
+## Enhancements
+
+#### Improved logging in case of invalid entry in Generic DVE Table [ID_38091]
+
+Logging has been improved in case there is an invalid entry in the *Generic DVE Table*. An error will be shown specifying which entry is invalid.
```

Comment:
can you precise this is when using SRM_DiscoverResources script ( used  to provision resources )?